x/concentrated-liquidity/swaps_test.go

```diff
@@ -2515,70 +2483,18 @@ func (s *KeeperTestSuite) TestCalcInAmtGivenOut_NonMutative() {
 			s.Require().Equal(1,
 				additiveFeeGrowthGlobalErrTolerance.CompareBigDec(
 					osmomath.BigDecFromSDKDec(test.expectedFeeGrowthAccumulatorValue),
-					osmomath.BigDecFromSDKDec(feeAccum.GetValue().AmountOf(test.tokenIn.Denom)),
+					osmomath.BigDecFromSDKDec(feeAccum.GetValue().AmountOf(test.tokenInDenom)),
 				),
 			)
-		})
-	}
-}
-
-// TestComputeInAmtGivenOut tests that ComputeInAmtGivenOut successfully performs state changes as expected.
-func (s *KeeperTestSuite) TestComputeInAmtGivenOut() {
```

Comment:
why removing this test? I think we should also test outGivenIn logic

Reply:
https://github.com/osmosis-labs/osmosis/pull/5207/files#diff-1a552fcc0a3d36556402345e840b658a7daccb101d7f73bca9ee2e3057ade2e6R2427 we have `inGivenOut` test here